utils/form-schema.ts

```diff
@@ -0,0 +1,24 @@
+import { z } from "zod";
+
```

Comment:
For consistency we could create reusable schemas for common fields like email
```suggestion
const emailSchema = z.string().email({ message: "Email address is invalid." });
```